flink-filesystems/flink-s3-fs-presto/src/main/resources/META-INF/NOTICE

```diff
@@ -57,3 +57,8 @@ This project bundles the following dependencies under BSD License (https://opens
 See bundled license files for details.
 
 - org.codehaus.woodstox:stax2-api:3.1.4 (https://github.com/FasterXML/stax2-api/tree/stax2-api-3.1.4)
+
+This project bundles the following dependencies under the CDDL 1.1 license.
+See bundled license files for details.
```

Comment:
somethings missing here ;)

you need to include [these](https://github.com/apache/flink/blob/master/flink-dist/src/main/resources/META-INF/licenses/LICENSE.jaxb) [two](https://github.com/apache/flink/blob/master/flink-dist/src/main/resources/META-INF/licenses/LICENSE.javax.activation) files under `resources/META-INF/licenses`.

Reply:
I already suspected that, but was not able to find them (in plain sight...).